### ISSUE TITLE
Give the option to the user to provide their own encryption key on startup

### ISSUE THREAD

**Issue by altinctrl** [CutFlame/MCDSaveEdit#39]
With the latest update to Minecraft Dungeons a new AES key is required to decrypt the pak files. By giving the user the option to provide their own key we remove the need to wait for a new release to use the tool and enable it to be used with older versions of the game.

**Comment by CutFlame** [comment]
When I build the releases I include all the AES keys for any of the releases of MCD so the latest version of MCDSaveEdit is already backwards compatible.

**Comment by Squoshy-Boop** [comment]
> When I build the releases I include all the AES keys for any of the releases of MCD so the latest version of MCDSaveEdit is already backwards compatible.

Yes, that's true, but for example now we have to wait for you to update, when we could just put the new one without bothering you.
